.circleci/config.yml

```diff
@@ -41,21 +47,18 @@ commands:
     description: "Add changes to SVN"
     steps:
       - run:
-          command: if [[ ! -z $(svn st | grep ^\!) ]]; then svn st | grep ^! | awk '{print " --force "$2}' | xargs -0r svn rm; fi
-      - run: svn add --force .
+          command: |
+            svn stat | { grep -E '^\?' || true; } | awk '{print $2}' | xargs -r svn add
+            svn stat | { grep -E '^\!' || true; } | awk '{print $2}' | xargs -r svn rm
+            echo "Here is the svn stat about to be checked in:"
+            svn stat --verbose
 
   svn_create_tag:
     description: "Create a SVN tag"
     steps:
-      - set_verision_variable
+      - set_version_variable
       - run: svn cp trunk tags/${VERSION}
```

Comment:
This [looks correct](https://svnbook.red-bean.com/nightly/en/svn.branchmerge.tags.html)